test/integration/master/kms_transformation_test.go

```diff
@@ -267,7 +267,7 @@ func mustBeHealthy(t *testing.T, checkName string, clientConfig *rest.Config) {
 	})
 
 	if pollErr == wait.ErrWaitTimeout {
```

Comment:
I do think we still need to assign `restErr = err` inside the loop above

Reply:
yeah，I agree，now i fixed it.